docs/csaf_aggregator.md

```diff
@@ -119,6 +119,16 @@ This can be configured for each entry, by the config option with the same name.
 If not given it is taken from the configured default
 Otherwise, the internal default "on best effort" is used.
 
+If a provider's `create_service_document` option is set to true,
+a `service.json` will be written listing its ROLIE feed documents.
+If it is not set or set to false, then no `service.json` will be written.
+
+To further dissect CSAF-documents by criteria like document category,
+languages or values of the branch category within the Product Tree you
+can set a providers categories value as an array of either
+literal categories or, by prepending "expr:", have it be read
+through JSONPath values.
```

Comment:
Please also add a link to the explanation from the [csaf_provider config](https://github.com/csaf-poc/csaf_distribution/blob/main/docs/csaf_provider.md); e.g. for more details on syntax cf. to ~

Reply:
Is the change done in https://github.com/csaf-poc/csaf_distribution/pull/333/commits/40366fa4970b4fa4dec49fd2d0fd69a03665693a sufficient?